Makefile

```diff
@@ -127,23 +127,23 @@ copyright: cmd/tools/copyright/licensegen.go
 
 cadence-cassandra-tool: $(TOOLS_SRC)
 	@echo "compiling cadence-cassandra-tool with OS: $(GOOS), ARCH: $(GOARCH)"
-	go build -o cadence-cassandra-tool cmd/tools/cassandra/main.go
+	go build -ldflags '-w' -o cadence-cassandra-tool cmd/tools/cassandra/main.go
```

Comment:
From that golang issue seems like this issue has been fixed in v1.15.4? Do we still need this change in this case?

Reply:
removed